fancy-falcons/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\Henry\\.virtualenvs\\fancy-falcons-wUudezNG\\Scripts\\python.exe"
+}
```

Comment:
The .vscode folder should not be comitted, this is for your local project settings